test-projects/project.visivo.yml

```diff
@@ -0,0 +1,79 @@
+# V2 Spec
+
+name: V2
+
+groups:
+  - name: Executives
+    emails:
+      - ceo@visivo.io
+      - cto@visivo.io
+
+conditions:
+  - name: if_during_business_hours
+    logic: if now() < 16:00 && now() > 08:00
+  - name: if_production
+    logic: if env == production
+  - name: if_morning_notifications
+    logic: if env["MORNING_NOTIFICATION"] == true
+
+alerts:
+  - name: Failure During Business Hours
+    condition: if_during_business_hours
+    message: |
+      Trigger {{ trigger.name }} has failed
```

Comment:
We could always do something like `{{ ... }}` for project references.  But `project(...)` would be more consistent.  The starting point will always need to be understood.  Relative to what point.

Reply:
I do like {{ }} because it stand out.

Reply:
- logic: assert_that( {{ Aggregated Line.props.marker.line.color[1] }} ).is_equal_to("#D25946")